hiking-demo/README.md

```diff
@@ -0,0 +1,15 @@
+# Simple CRUD Demo
+
+A simple CRUD application using Java EE (JPA, JAX-RS, CDI etc.), based on top of WildFly and MySQL.
+Used as an example for streaming changes out of a database using Debezium.
+
+To run the app, follow these steps:
+
+    mvn clean package
+    docker build --no-cache -t debezium-examples/hike-manager:latest -f Dockerfile .
+
+    docker run -it --rm --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw debezium/example-mysql:0.7
+
+    docker run -it --rm -p 8080:8080 --link mysql debezium-examples/hike-manager:latest
+
+Then visit the application in a browser at http://localhost:8080/hikr-1.0-SNAPSHOT/hikes.html.
```

Comment:
I see that instructions are related to use Docker on a local environment, should be change them in order to describe how to run the demo but on OpenShift and using the already deployed Kafka cluster ? Is that something that you want to do later after merging this PR ?

Reply:
It's something we definitely should do. I just gave the local Docker instructions as a starting point. I wanted to wait for the first round of your instructions to set up Kafka itself. Then we can agree how this should be done, using s2i or similar.

Reply:
so are you saying that you'd prefer having this merged and then updating the instructions about using OpenShift instead of just Docker ?